core/bourbon/settings/_settings.scss

```diff
@@ -0,0 +1,43 @@
+@charset "UTF-8";
+
+/// Default Bourbon configuration settings.
+///
+/// @type map
+///
+/// @property {string} global-font-file-formats [ttf woff2 woff]
+///   Global font file formats for the `font-face` mixin.
+///
+/// @property {number (with unit)} modular-scale.base [1em]
+///   Global base value for the `modular-scale` function
+///
+/// @property {number (unitless)} modular-scale.ratio [$major-third (1.25)]
+///   Global base ratio for the `modular-scale` function
+///
+/// @property {bool} rails-asset-pipeline [false]
+///   Enable or disable the `$asset-pipeline` variable for all functions that
+///   accept it.
+
+$bourbon-defaults: (
+  "global-font-file-formats": ttf woff2 woff,
+  "modular-scale": (
+    "base": 1em,
+    "ratio": $major-third,
+  ),
+  "rails-asset-pipeline": false,
+);
+
+/// User overrides of Bourbon configuration settings
+///
+/// @type map
```

Comment:
Yes, these are SassDoc comments. This particular line is indicating that this variable is a map.